### PR TITLE
Add examples for endswith and startswith

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

**Comment by simonbyrne** [comment]
Looks like the spacing is a bit off in the doctest

File: base/strings/util.jl [hunk 1]
```diff
@@ -109,6 +109,17 @@ used to implement specialized methods.
 !!! compat "Julia 1.5"
     The single argument `endswith(suffix)` requires at least Julia 1.5.
 
+# Examples
+```jldoctest
+julia> endswith_julia = endswith("Julia")
+(::Base.Fix2{typeof(endswith),String}) (generic function with 1 method)
```
Comment (KristofferC) [inline base/strings/util.jl:115]: Maybe put a semicolon after this to avoid showing this really ugly type.

Comment (logankilpatrick) [reply base/strings/util.jl]: Done

**Comment by logankilpatrick** [comment]
Should be good now @simonbyrne, I removed the part that was giving the error per @KristofferC's suggestion.

**Review by simonbyrne — APPROVED** [review]

**Pull request merged**